<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APTSourceEdit</class>
 <widget class="QDialog" name="APTSourceEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>APT Source Edit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="horizontalSpacing">
    <number>15</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="1" column="1">
    <widget class="QRadioButton" name="radioBinary">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Binary</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">debTypeGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>347</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="txtUri">
     <property name="placeholderText">
      <string>URI</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="title">
     <property name="text">
      <string>APT Source</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="errorMsg">
     <property name="text">
      <string>Fields cannot be left blank. </string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLineEdit" name="txtComponents">
     <property name="placeholderText">
      <string>Components</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="saveBtn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLineEdit" name="txtDistribution">
     <property name="placeholderText">
      <string>Distribution</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioSource">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Source</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">debTypeGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLineEdit" name="txtOptions">
     <property name="placeholderText">
      <string>Options</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtOptions</tabstop>
  <tabstop>txtUri</tabstop>
  <tabstop>txtDistribution</tabstop>
  <tabstop>txtComponents</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="debTypeGroup"/>
 </buttongroups>
</ui>
